api/apps/v1alpha1/common_types.go

```diff
@@ -47,9 +48,10 @@ type Metrics struct {
 
 // ServiceMonitor defines attributes to create a service monitor
 type ServiceMonitor struct {
-	AdditionalLabels map[string]string `json:"additionalLabels,omitempty"`
-	Interval         promv1.Duration   `json:"interval,omitempty"`
-	ScrapeTimeout    promv1.Duration   `json:"scrapeTimeout,omitempty"`
+	AdditionalLabels      map[string]string `json:"additionalLabels,omitempty"`
+	AdditionalAnnotations map[string]string `json:"additionalAnnotations,omitempty"`
```

Comment:
same here, just "Annotations" should be used.

Reply:
Fixed